pkg/api/server/register_images.go

```diff
@@ -1615,5 +1615,39 @@ func (s *APIServer) registerImagesHandlers(r *mux.Router) error {
 	//   500:
 	//     $ref: "#/responses/internalError"
 	r.Handle(VersionedPath("/libpod/build"), s.APIHandler(compat.BuildImage)).Methods(http.MethodPost)
+
+	// swagger:operation POST /libpod/images/scp libpod ImageScpLibpod
+	// ---
+	// tags:
+	//  - images
+	// summary: Copy an image from one host to another
+	// description: Copy an image from one host to another
+	// parameters:
+	//   - in: query
+	//     name: source
+	//     required: true
+	//     description: source connection/image
+	//     type: string
```

Comment:
```suggestion
	// swagger:operation POST /libpod/images/scp/{name} libpod ImageScpLibpod
	// ---
	// tags:
	//  - images
	// summary: Copy an image from one host to another
	// description: Copy an image from one host to another
	// parameters:
	//   - in: path
	//     name: name
	//     required: true
	//     description: source connection/image
	//     type: string
```

Reply:
I will put this in the path but keep it as source as it can be either an image or a ssh destination (and the native docs uses source/dest lingo)

Reply:
oh do I have to call this name in order to put it in the path/pull it out of the path @jwhonce ?